packages/catalog-process/src/services/catalogService.ts

```diff
@@ -20,6 +24,7 @@ import {
   operationForbidden,
   ListResult,
   WithMetadata,
+  toBinaryData,
```

Comment:
This function is defined in `models` package, at the moment handles only `EServiceEvent`, but in future we will need to a specific binary converter for other models like `agreement`. 
We can consider to export this function with a more specific name for a catalog's models?  
The suggested improvement is not strictly related to this PR, we can handle it in future.